_posts/2018-05-16-strimzi-0-4-0-release.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Strimzi 0.4.0 released!"
+date: 2018-04-21
```

Comment:
Should we add also author name here?

Reply:
I have never added an author in the previous blog posts as well. I was thinking about a sort of author "Strimzi team" but it drives me to think about a closed team developing Strimzi which is not true.